src/ccutil/unicharset.cpp

```diff
@@ -960,15 +959,16 @@ void UNICHARSET::post_load_setup() {
 
   // Compute default script. Use the highest-counting alpha script, that is
   // not the common script, as that still contains some "alphas".
-  int *script_counts = new int[script_table_size_used];
-  memset(script_counts, 0, sizeof(*script_counts) * script_table_size_used);
+  int script_table_size = static_cast<int>(script_names_.size());
```

Comment:
Maybe using size_t instead of int would be better and make the type cast unnecessary.